css/style.css

```diff
@@ -1,3 +1,109 @@
 h1 {
-  color: salmon;
+  color: #424647;
+  text-align: center;
+}
+
+p {
+  color: #424647;
+  text-align:center;
+  margin-top: -15px;
+}
+
+body {
+  background-color: #E9EBE7;
+  font-family: 'Fira Sans Condensed', sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.images {
+  color: white;
+  font-size: 1.75em;
+  font-weight: bold;
+}
+
+.SF1 {
+  background-image:url(../img/san-francisco.jpg);
+  background-size: cover;
+  width: 250px;
+  height: 250px;
+  float: left;
+  text-align: center;
+  vertical-align: middle;
+  line-height: 250px;
+  margin: 10px;
+}
+
+.NY {
+  background-image:url(../img/new-york.jpg);
+  background-size: cover;
+  width: 250px;
+  height: 250px;
+  float: left;
+  text-align: center;
+  vertical-align: middle;
```

Comment:
Not sure how you intended to use `vertical-align`, but it is a commonly misunderstood property. It's used to align items along a line -- common use cases are making super- or sub-script text. It is **not** used to vertically align entire elements on a page.

http://phrogz.net/css/vertical-align/